drv/sidecar-front-io/sidecar_qsfp_x32_controller.adoc

```diff
@@ -126,56 +127,61 @@ Control bits related to VSC8562  (valid on FPGA1 only)
 [cols=4,options="header"]
 |===
 | Bits | SW Access | Name | Function
-|[7:2] |  | - | Reserved
+|[7:3] |  | - | Reserved
+|[2] | rw | CLEAR_POWER_FAULT | Setting this bit to 1 will clear the timed out state of the
+V1P0 and V2P5 rail controllers, allowing PHY power
+sequencing to be attempted again.
 |[1] | rw | COMA_MODE | Software control for the COMA_MODE pin.
 |[0] | rw | EN | Enable for the VSC8562 power up state machine
 |===
 
-Status bits for the SMI interface to the VSC8562 (valid on FPGA1 only)
+Status bits for the SMI interface to the VSC8562 (valid on
```

Comment:
Did we change something in how our asciidoc files are generated?  This is weirdly split into two lines with funky intendation.

Reply:
No, this is because of the RDL source file used to generate everything. I will fix that.